target/linux/ramips/image/mt7621.mk

```diff
@@ -610,6 +610,15 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hilink_hlk-7621a-evb
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  DEVICE_VENDOR := HiLink
+  DEVICE_MODEL := HLK-7621A evaluation board
+  IMAGE_SIZE := 32448k
+endef
```

Comment:
No wifi device packages?

Reply:
Ok, so I can add here also the device packages for the wifi, yes. I have assigned them manually in menu config :). Thanks for pointing this out. Will add:

DEVICE_PACKAGES += kmod-mt76x2 kmod-usb3